Dockerfile

```diff
@@ -178,11 +178,12 @@ COPY rootfs/rootfs $ROOTFS
 
 # Build the Hyper-V KVP Daemon
 RUN cd /linux-kernel && \
-    make headers_install INSTALL_HDR_PATH=/usr && \
+    make INSTALL_HDR_PATH=/tmp/kheaders headers_install && \
     cd /linux-kernel/tools/hv && \
-    sed -i 's/\(^CFLAGS = .*\)/\1 -m32/' Makefile && \
+    sed -i 's/\(^CFLAGS = .*\)/\1 -m32 -I\/tmp\/kheaders\/include/' Makefile && \
```

Comment:
Wouldn't this be cleaner/easier to read if we switched the replacement separator?
ie:
```Dockerfile
    sed -i 's!\(^CFLAGS = .*\)!\1 -m32 -I/tmp/kheaders/include!' Makefile && \
```